music_assistant/server/helpers/compare.py

```diff
@@ -112,6 +112,9 @@ def compare_track(
     # return early on exact item_id match
     if compare_item_ids(base_item, compare_item):
         return True
+    # return early on MBID match to avoid issues with ISRC (https://github.com/music-assistant/hass-music-assistant/issues/2316)
```

Comment:
```suggestion
    # return early on MBID match to avoid issues with ISRC 
    # (https://github.com/music-assistant/hass-music-assistant/issues/2316)
```